documentation/writing_yaml_plans.md

```diff
@@ -77,40 +77,37 @@ Each plan name segment must begin with a lowercase letter and:
 
 ## Plan structure
 
-YAML plans contain a list of steps with optional parameters and results.
+YAML plans contain a list of steps and can include additional optional keys.
+The following top-level keys are available:
```

Comment:
These formatting improvements are SO MUCH BETTER! I feel like at some point we'll want to add a reference doc for YAML plan keys, but definitely doesn't have to be here (I'm also like 99% sure you've had the same idea :stuck_out_tongue_winking_eye:)

Reply:
You know me so well 😅